cmd/oci/oci.go

```diff
@@ -37,6 +45,9 @@ var OCICmd = &cobra.Command{
 	bsf oci <environment name> 
 	bsf oci <environment name> --platform <platform>
 	bsf oci <environment name> --platform <platform> --output <output directory>
+
+	bsf oci pkgs.runtime --name=<image name> --tag=<image tag>
```

Comment:
Name should be picked from hcl param..
`dev` should be done with `--dev` flag. 
By default, `bsf oci pkgs` should do runtime.